AdyenDropIn/DropInComponent.swift

```diff
@@ -229,43 +225,95 @@ public final class DropInComponent: NSObject,
         return handler
     }()
 
-    internal func paymentMethodListComponent(onCancel: (() -> Void)?) -> PaymentMethodListComponent {
-        let paymentComponents = componentManager.sections
-        let component = PaymentMethodListComponent(
+    // ================= ROOT VIEW CONTROLLER ===============
+
+    internal lazy var rootViewController: UIViewController = {
```

Comment:
This is quite hard to understand and it contains a lot of logic. Would be nice to simplify and move decisions out of here in the future PRs.